### DISCUSSION TITLE
0.9.3을 사용하다 0.10.3으로 업데이트 한 후

### DISCUSSION THREAD

**Discussion by nowtoday** [bab2min/kiwipiepy#67] [Q&A]
pyinstaller로 컴파일해서 쓰는 것을 병행하고 있는데요...
기존에는 kiwi = Kiwi(model_path='./') 정의해서 컴파일 후 모델파일들을 실행파일과 같은 폴더에 넣고서 사용이 가능했는데
지금은 불가능해지네요... 

load_default_dict=load_default_dict,
RuntimeError: 'KIWI' is needed but 'IWIK': iostream stream error 

설정을 빼보기도 하고
model_path=''
model_path='.'
model_path='./.'
등으로 해봤는데 

그냥 스크립트로 실행하는 것과 컴파일 해서 실행하는 것을 모두 만족할 수 있는 방법이
model_path='./model'
이것 밖에 없는 것 같아서...
혹시나 같은 폴더에 넣고 컴파일과 스크립트 실행을 다 가능하게 할 방법이 있을지 해서...

**Comment by bab2min** [comment]
@nowtoday 안녕하세요
`RuntimeError: 'KIWI' is needed but 'IWIK': iostream stream error`
이 오류로 볼때 모델 파일을 못 읽고 있는 문제는 아닌걸로 보이구요, 엔디안 관련해서 버그가 있는듯합니다.
혹시 해당 오류가 발생하는 코드나 실행파일을 공유해주실수 있으실까요?

**Reply by nowtoday** [reply]
혹시나 해서 간단한 소스로 만들어 작업해 봤더니 정상 작동하네요...
제가 앞서 만든 다른 루틴 소스들과의 문제가 있는 것은 아닌지 찾아보고 있습니다...
(한편으로는 그냥 ./model 폴더에 다 몰아 넣어서 작동시키는게 편해지기도 했네요 ^^;;)

**Reply by nowtoday** [reply]
아예 다른 얘기이긴 한데...
혹시 kiwipiepy 버전을 보는 방법이 kiwipiepy.version 으로 알고 있는데 (조만간 kiwipiepy.__version__ 으로 바뀐다고 적혀있고...)
하지만 print로 보면 property object at 0x00000241A1D63AE0 로만 나오던데...
Tomotopy는 간단하게 버전 확인이 가능하지만 kiwipiepy 만 달리 나와서....  현재 버전인 0.11.0 에서도...

**Reply by bab2min** [reply]
`__version__`을 쓰시면 됩니다.
```python
>>> import kiwipiepy
>>> kiwipiepy.__version__
'0.11.0'
```

**Reply by nowtoday** [reply]
별도 import를 한번 더 해야겠군요...
from kiwipiepy import Kiwi로 해서
Kiwi.__version__ 이 안나왔었나보군요 ^^;;
